cmd/osquery-perf/agent.go

```diff
@@ -804,12 +829,35 @@ func (a *agent) config() {
 	}
 
 	var scheduledQueries []string
+	var scheduledQueryData []scheduledQuery
 	for packName, pack := range parsedResp.Packs {
-		for queryName := range pack.Queries {
+		for queryName, query := range pack.Queries {
 			scheduledQueries = append(scheduledQueries, packName+"_"+queryName)
```

Comment:
You could grab `packName` and put it in the `scheduledQuery`. Then pass that to `a.SubmitLogs`. Which would use it by setting the query name as `pack/$packName/$queryName` instead of hardcoding `pack/Global/$queryName`.

Reply:
Good idea! done